### PR TITLE
Added option to cancel auto-authorize

### LINKED ISSUE #44: Be able to stop scheldueRenewal in Login.authorize

### PR DESCRIPTION
**Pull request by ghost**
```js
const authResult = await Login.authorize({...});
Login.stopAutoAuthorize(authResult);
```

Fixes #44

### CODE REVIEW

File: src/Login.ts [hunk 1]
```diff
@@ -104,6 +104,14 @@ export class Login {
     return url;
   }
 
+  public static stopAutoAuthorize(authResult: AuthResult): void {
```
Comment (valgussev) [inline src/Login.ts:107]: Why not just accessToken?

Comment (ghost) [reply src/Login.ts:107]: Yes I could but then we can't use WeakMap but maybe that is ok?

Comment (ghost) [reply src/Login.ts:107]: I will take a look

**Comment by ghost** [comment]
Closed in favour of #54

**Pull request closed without merging**